src/main/java/org/cru/redegg/recording/impl/DefaultWebErrorRecorder.java

```diff
@@ -137,7 +142,9 @@ public WebErrorRecorder recordRequestPostParameters(Multimap<String, String> pos
     @Override
     public WebErrorRecorder recordEntityRepresentation(String entityRepresentation) {
         checkState(!completed);
-        webContext.setEntityRepresentation(checkNotNull(entityRepresentation));
+        checkNotNull(entityRepresentation);
+        String sanitized = entitySanitizer.sanitizeEntity(entityRepresentation);
+        webContext.setEntityRepresentation(sanitized);
```

Comment:
Regarding `DefaultWebErrorRecorder`, line 161 in this PR, I realize there is no change to this code in this PR, but I'm confused that you should be doing `checkState(!completed)` in a method where you are setting the completed state. Can you clarify for me?